pyproject.toml

```diff
@@ -87,20 +85,13 @@ dev = [
 [project.urls]
 repository    = "https://github.com/citadel-ai/langcheck"
 
-[tool.yapf]
-based_on_style = "google"
-column_limit = 80
-
-[tool.yapfignore]
-ignore_patterns = [
-    "venv/*",
+[tool.ruff]
+lint.select = ["E4", "E7", "E9", "F", "I"]
```

Comment:
We also include "Q" in the Python client

Reply:
Good catch, added!!